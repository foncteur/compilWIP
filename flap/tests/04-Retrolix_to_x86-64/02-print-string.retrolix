external printf

globals ()
  l00: %rdi <- copy "Hello world!\n";
  l01: %rax <- copy 0;
  l02: printf();
  l03: exit;
end