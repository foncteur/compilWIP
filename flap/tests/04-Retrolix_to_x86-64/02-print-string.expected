Process exited with status exited(0).
STDOUT:
Hello world!

STDERR:



